### PR TITLE
Adding App Store Connect texts as DocC article #124

### LINKED ISSUE #124: FEATURE  Create App Store listing in App Store Connect
Set up app name, subtitle, keywords, description and all other fields in App Store Connect.

TODO: expand this listing including suggested text for all fields.

### PR DESCRIPTION
**Pull request by SuzGupta**
…re listing

# What it Does
* Closes #124 
* Adds suggested name, subtitle, description, keywords and some other settings to copy and paste into App Store Connect when the maintainer is ready to submit the app

# How I Tested
* Choose Build Documentation from Xcode's Product menu
* Review the result

# Notes
* There's more that could be said about App Store Connect, but I think this is a solid start.

# Screenshot
AppStoreListingArticle

### CODE REVIEW

**Comment by SuzGupta** [comment]
Blergh, that was two more commits than it needed to be, but I think the actual text is fairly solid.

File: Basic-Car-Maintenance/Documentation.docc/App Store Listing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Overview
+
+This article contains suggested texts and settings for the App Store listing, which will be set up in App Store Connect after Hacktoberfest.
+
+Here's the Apple developer site covering how to set up an app's product page, as well as the many other things you can do with App Store Connect: https://developer.apple.com/app-store-connect/
+
+### App Information
+
+Set up the app's name and subtitle under **General** ▸ **App Information** in the **Localizable Information** section at the top of the page. You have a maximum of 30 characters for each field. According to [Appfigures](https://appfigures.com/resources/guides/app-name-optimization), you should portion out keywords between the name and the subtitle for optimal ASO, App Store Optimization, which is the black art of doing everything you can in your App Store listing to make it highly visible in the Store.
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Documentation.docc/App Store Listing.md]:
```suggestion
Set up the app's name and subtitle under **General** ▸ **App Information** in the **Localizable Information** section at the top of the page. You have a maximum of 30 characters for each field. According to [Appfigures](https://appfigures.com/resources/guides/app-name-optimization), you should portion out keywords between the name and the subtitle for optimal ASO, App Store Optimization, which is the magic of doing everything you can in your App Store listing to make it highly visible in the Store.
```

File: Basic-Car-Maintenance/Documentation.docc/App Store Listing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- **Primary Language** is set in the build and will be `English (U.S.)` for this app.
+- **Category** Looking at potential competitors for this app, there's no clear winner as to category. `Utilities` could be a logical choice, as could `Lifestyle`. At the time of this writing, Apple currently shows Lifestyle in Top Categories (without needed to tap See All) although it does not show Utilities, so this suggests we should pick `Lifestyle`. It's not clear whether there's a benefit to setting the optional Secondary category, so let's leave this unset.
+
+### Version Information
+Enter the following metadata on the version information page, which appears under **iOS App** at the top left-hand side of the App Store tab in App Store Connect
+
+#### Description
+`Basic Auto Vehicle Maintenance lets you easily log repairs and routine service. You can also track odometer readings, handy for business travel or just to figure out how far your fuel gets you.`
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Documentation.docc/App Store Listing.md]: Can we also word this in a way that accommodates electric vehicles?

File: Basic-Car-Maintenance/Documentation.docc/App Store Listing.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+`This app began as a Hacktoberfest 2023 project and remains open source, maintained by Mikaela Caron. Look for mikaelacaron / Basic-Car-Maintenance and come contribute!`
+
+#### Keywords
+You have 100 characters of keyword space, which get combined with the Name and Subtitle fields discussed above when a user searches the App Store. Ariel Michaeli of Appfigures recently advised to put the more important keywords closer to the start of this field. Separate keywords with commas; omit spaces. 
+
+Suggested: `car,van,motorcycle,bike,care,repair,manage,easy,simple,oil,change,tire,rotation,record,fuel,gas,log`
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Documentation.docc/App Store Listing.md]: Also some kind of electric vehicles words be great!

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
Love this! Thanks for being so detailed!
I have a few suggestions about also including verbiage for electric vehicles and hybrids

Once you make those changes, click "Resolve" to resolve the comment I made. If you have any questions you can comment directly on that thread.

After you're done, click the re-review button

image

File: Basic-Car-Maintenance/Documentation.docc/App Store Listing.md (hunk 3 above)
Comment (SuzGupta) [reply Basic-Car-Maintenance/Documentation.docc/App Store Listing.md]: will remove `motorcycle` and `log` and add `electric` and `charge`, leaving us at exactly 100 character

File: Basic-Car-Maintenance/Documentation.docc/App Store Listing.md (hunk 2 above)
Comment (SuzGupta) [reply Basic-Car-Maintenance/Documentation.docc/App Store Listing.md]: will change that last bit to read "how far your fuel or battery charge gets you"

File: Basic-Car-Maintenance/Documentation.docc/App Store Listing.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# App Store Listing
+
+## Overview
+
+This article contains suggested texts and settings for the App Store listing, which will be set up in App Store Connect after Hacktoberfest.
+
+Here's the Apple developer site covering how to set up an app's product page, as well as the many other things you can do with App Store Connect: https://developer.apple.com/app-store-connect/
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Documentation.docc/App Store Listing.md]:
```suggestion
Here's the Apple developer site covering how to set up an app's product page, as well as the many other things you can do with App Store Connect: [https://developer.apple.com/app-store-connect/](https://developer.apple.com/app-store-connect/)
```

Makes this into a hyperlink

**Review by mikaelacaron — APPROVED** [review]
Awesome!! Thanks for being so detailed! I have one small suggestion that I'll commit directly

**Pull request merged**